content/blog/2016/2016-01-27-jenkins-world-call-for-papers.adoc

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Jenkins World 2016: Call For Papers Is Open!"
+tags:
+- jenkins world
+- event
+- jenkins
+author: alyssat
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016,180,float="right",link=" https://jenkins-cfp.herokuapp.com/events/jenkins-world-2016"]
+
+Planning is underway for the annual Jenkins User Conference. Conference dates are September 13 - 15, 2016, at the Santa Clara Convention Center in California.  In previous years, Jenkins User Conference has been a multi-city conference. With year after year growth, the Jenkins User Conference has proven that Jenkins is most popular open source automation server.  That being said, Jenkins User Conference will be Jenkins World in 2016. It will be a one city, three days, one global conference.
+ 
+Jenkins World 2016 will bring together key Jenkins contributors, and practitioners from around the globe to unite in one location. The ultimate goal is to provide a platform for discussions, sharing, and hands-on learning for beginners, intermediate to advance users.
+
+This conference like past Jenkins User Conferences, has been sponsored by link:https://www.cloudbees.com[CloudBees] (the premier sponsor), is an essential gathering for any developers interested continuous delivery automation.

```

Comment:
great point. i'll tweak accordingly. Thanks Oleg.

On Wed, Jan 27, 2016 at 10:27 AM, Oleg Nenashev <notifications@github.com>
wrote:

> In content/blog/2016/2016-01-27-jenkins-world-call-for-papers.adoc
> <https://github.com/jenkins-infra/jenkins.io/pull/58#discussion_r51026605>
> :
>
> > +layout: post
> > +title: "Jenkins World 2016: Call For Papers Is Open!"
> > +tags:
> > +- jenkins world
> > +- event
> > +- jenkins
> > +author: alyssat
> > +---
> > +
> > +image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016,180,float="right",link=" https://jenkins-cfp.herokuapp.com/events/jenkins-world-2016"]
> > +
> > +Planning is underway for the annual Jenkins User Conference. Conference dates are September 13 - 15, 2016, at the Santa Clara Convention Center in California.  In previous years, Jenkins User Conference has been a multi-city conference. With year after year growth, the Jenkins User Conference has proven that Jenkins is most popular open source automation server.  That being said, Jenkins User Conference will be Jenkins World in 2016. It will be a one city, three days, one global conference.
> > +
> > +Jenkins World 2016 will bring together key Jenkins contributors, and practitioners from around the globe to unite in one location. The ultimate goal is to provide a platform for discussions, sharing, and hands-on learning for beginners, intermediate to advance users.
> > +
> > +This conference like past Jenkins User Conferences, has been sponsored by link:https://www.cloudbees.com[CloudBees] (the premier sponsor), is an essential gathering for any developers interested continuous delivery automation.
>
> So "CD" is the main focus. Would worth to mention other areas in the blog
> in order to avoid the "CD-only" impression
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/58/files#r51026605>.
>